chia/util/cached_bls.py

```diff
@@ -42,12 +42,12 @@ def get_pairings(
                 pk_parsed = G1Element.from_bytes(pks[i])
                 pk_bytes_to_g1[pks[i]] = pk_parsed
 
-            pairing = pk_parsed.pair(aug_hash)
+            pairing = aug_hash.pair(pk_parsed)
```

Comment:
`pair()` is commutative. this unifies the member function we use to be called on `G2Element`